### DISCUSSION TITLE
failed to create zip:  TypeError [ERR_INVALID_ARG_TYPE]: The "start" argument must be of type number. Received null

### DISCUSSION THREAD

**Discussion by tnvu** [emmercm/igir#983] [Q&A] [answered]
I have some roms erroring when I am trying "igir move zip".

 I don't know why this is happening. I tested the source zips and they seem fine, but it seems like there is some problem with the zip code in igir?

Has anyone seen this before and come up with a fix or workaround?

**Comment by emmercm** [comment]
This sounds like a legitimate bug. Please fill out a complete [bug report](https://github.com/emmercm/igir/issues/new/choose), including trace logs.

**Reply by emmercm** [reply, marked as answer]
https://github.com/emmercm/igir/issues/991
